docs/admin/groups.md

```diff
@@ -0,0 +1,142 @@
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_2.png" alt="drawing" width="500"/> |
+
+Click on "Group management".
+
+|                           Group management console                               |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_3.png" alt="drawing" width="500"/> |
+
+
+Click on the "+" button. 
+
+Add group details including the ID (which must be unique), display name and a group description. 
+
+|                              Add Group Dialog                                    |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_4.png" alt="drawing" width="500"/> |
+
+
+Click on the "Submit" button to add the group. The group will now appear in the Group Management section of the Admin Console.
+
+|                          Group added and listed                                  |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_5.png" alt="drawing" width="500"/> |
+
+## Add users to a group
+
+Select the group in the Group Management section of the Admin Console. Once selected, click on the "Add member" button. 
+
+|                             Add member in Group admin console                    |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_6.png" alt="drawing" width="500"/> |
+
+
+Add details for the user to add. The username must match with the user to be added. You can fill in the other fields optionally. In future, this information may be retrieved from the Provena server if known. For now, this is manually entered/updated.
+
+|                           Add user to group  dialog                              |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_12.png" alt="drawing" width="500"/> |
+
+The user will be listed in the Users table in the Group admin page.
+
+|                            User added to group                                   |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_13.png" alt="drawing" width="500"/> |
+
+Additional users can be added with this flow. The figure below shows an update list of users in the 
+Users table in the Group admin page.
+
+|                         Other users added to group                               |
+| :------------------------------------------------------------------------------: |
+| <img src="../assets/images/admin/admin_group_14.png" alt="drawing" width="500"/> |
+
+
+## Manage group
```

Comment:
This heading is the same size as the one below it. Can it be changed to "# Manage groups".
And then above, "# Create/Manage Groups" can just be "#Create Groups" or "#Creating Groups with Users"?